src/containers/Playlist.js

```diff
@@ -35,7 +49,7 @@ class Playlist extends Component {
     const { currentOpenPlaylist, isCurrentPlaylistLoaded} = this.props
     if (isCurrentPlaylistLoaded) {
       return (
-        <div className='w-100 min-vh-100'>
+        <div className={'w-100 min-vh-100 '}>
```

Comment:
#nitpick don't think this has to be interpolated

Reply:
brackets are necessary if `'` is used instead of `"`